### ISSUE TITLE
Ensure ARIA roles, states, and properties are valid

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2491]
OCIO occurance 96%

The Aria tab on this sheet details this issue from the OCIO report: https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR.gov/508/onrr.gov%20website%20accessibility/OCIO%20report%20violations%20-%20Feb%20%2723.xlsx?d=w3e1f0b4a745244dc8d2aa85991bb45a3&csf=1&web=1&e=ysQzl7

This has been repored by lighthouse and the OCIO report. There are a variety of aria issues affecting every page. The OCIO report places each of the problems into one or more of the following categories: 

- The aria-expanded attribute is not allowed on this DIV

- The aria-level attribute is not allowed on this DIV

- This DIV has an aria-level attribute of , which is not an integer value greater than, or equal to, 1

- This INPUT has a mismatch between the value of its readonly and aria-readonly attributes

Lighthouse lists these 4 aria issues for every page:
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/d4a09d4d-22c3-45b9-bf1e-766e01d68030)

And this one for many pages, but not all:
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/59aa52f9-231b-4c11-a1f4-90d207f9db21)

More details and the pages affected are found in the section 508 folder in teams.

**Comment by greensomes** [comment]
Lists of links on the homepage for Reporter Letters have a role hierarchy of `list`, `listbox`, then `option`. However, according to Mozilla documentation, elements with role `list` should have children with role `listitem` and `listbox` should have elements with role `option`. Determining if the parent role should be `list` or `listbox`. See [Nielson Norman guidance](https://www.nngroup.com/articles/listbox-dropdown/) for more on listbox.

**Comment by greensomes** [comment]
As it turns out, the list issue above is governed by vuetify components. This has been addressed.

After committing changes and merging to dev, the Lighthouse score for the home page is now 100. Yay!
